### PR TITLE
Typo fix

### PR DESCRIPTION
**Pull request by DenrizSusam**
**What**:

**Why**:

**How**:

**Checklist**:

- [ ] Documentation added to the
      [docs site](https://github.com/testing-library/testing-library-docs)
- [ ] Tests
- [ ] Typescript definitions updated
- [x] Ready to be merged

### CODE REVIEW

**Comment by nickmccurdy** [comment]
I'm fine with merging this, but I don't remember how to properly make a semantic commit without publishing a new version

**Comment by marcosvega91** [comment]
I think that you can merge with a commit of type style

**Comment by kentcdodds** [comment]
We don't want a release for this, otherwise it doesn't matter much. https://www.conventionalcommits.org/en/v1.0.0/#specification

File: src/role-helpers.js [hunk 1]
```diff
@@ -154,7 +154,7 @@ function getRoles(container, {hidden = false} = {}) {
 
 function prettyRoles(dom, {hidden}) {
   const roles = getRoles(dom, {hidden})
-  //We prefer to skip generic role, we don't reccomand it
+  //We prefer to skip generic role, we don't recommend it
```
Comment (kentcdodds) [inline src/role-helpers.js:157]:
```suggestion
  // We prefer to skip generic role, we don't recommend it
```

**Comment by kentcdodds** [comment]
@all-contributors please add @DenrizSusam for code

**Pull request merged**